.dev/assets/shared/css/amp/amp.css

```diff
@@ -0,0 +1,28 @@
+body[class*="amp"] .primary-menu .menu-item-has-children:hover > .sub-menu {
+	display: none;
+}
```

Comment:
Why is this being hidden when rendered as AMP?

Reply:
@westonruter This is hidden on `:hover` since the default functionality is to 'click' the top level menu to display the sub menu.

Reply:
Why would that be any different in AMP? AMP pages are just webpages and are accessed in mobile or desktop, touch or mouse. This is especially so in Standard mode when there is only one version of your site, what the AMP team calls an “AMP-first” approach. See some examples: https://amp-wp.org/showcases/?template_mode=native

Reply:
The Transitional mode and Reader mode are examples of “paired AMP” which generally would only be served explicitly to mobile users when they encounter content on search. But this is not always the case. 

For example, if you install Google News as a PWA, articles you click on will be AMP pages. So they need to work in desktop as well as mobile:

![image](https://user-images.githubusercontent.com/134745/73110998-6e63d980-3ebd-11ea-833d-a078a351744f.png)